tests/foreman/cli/test_host.py

```diff
@@ -161,17 +162,15 @@ def test_positive_create_with_name(self):
             with self.subTest(name):
                 host = entities.Host()
                 host.create_missing()
-                result = Host.create({
+                result = make_host({
                     u'architecture-id': host.architecture.id,
                     u'domain-id': host.domain.id,
                     u'environment-id': host.environment.id,
-                    # pylint:disable=no-member
```

Comment:
I'm curious why you removed such comments? It's not like i like them or really care about pylint score, but still, these lines were valid as pylint actually can't see `host.organization` and `host.location`.